detox/local-cli/utils/frameworkUtils.js

```diff
@@ -0,0 +1,78 @@
+const os = require('os');
+const path = require('path');
+
+const exec = require('child-process-promise').exec;
+const fs = require('fs-extra');
+const _ = require('lodash');
+
+const detox = require('../../internals');
+
+const frameworkBuildScript = '../../scripts/build_local_framework.ios.sh';
+const xcuitestBuildScript = '../../scripts/build_local_xcuitest.ios.sh';
+
+const getFrameworkPath = _.once(() => path.join(os.homedir(), '/Library/Detox/ios/framework'));
+const getXcuitestPath = _.once(() => path.join(os.homedir(), '/Library/Detox/ios/xcuitest-runner'));
```

Comment:
@asafkorem  I wonder, can't that be taken from `src/utils/environment` ?

Reply:
I'll expose something from there.

Reply:
done